packages/govuk-frontend-review/src/stylesheets/app.scss

```diff
@@ -1,4 +1,5 @@
 $govuk-show-breakpoints: true;
+$govuk-new-typography-scale: true;
```

Comment:
If you want a nice CHANGELOG entry, the one for `$govuk-new-link-styles` was pretty good:
https://github.com/alphagov/govuk-frontend/releases/tag/v3.12.0

Maybe flag the Review app to `true` in a separate commit?

Reply:
Whoops I meant to do this but forgot!